src/main/java/com/bandwidth/sdk/messaging/MessagingClient.java

```diff
@@ -296,4 +301,68 @@ public void downloadMessageMediaToFile(String mediaUrl, String path) {
             return (Void) null;
         });
     }
+
+    /**
+     * Lists MMS media content.
+     *
+     * @return URL that can be sent in an MMS
+     */
+    public List<Media> listMedia() {
+        return listMediaAsync().join();
+    }
+
+    /**
+     * Lists MMS media content.
+     *
+     * @return CompletableFuture that contains URL that can be sent in an MMS
+     */
+    public CompletableFuture<List<Media>> listMediaAsync() {
```

Comment:
Could we accept an optional continuation token here, and the return type be an object that contains 1 page of `List<Media>`, and the continuation header? Then the customer can loop through the pages at their own pace instead of receiving an unbounded number of media results back

Reply:
You can also add a `next()` method on the return object that easily fetches the next page